python/pyspark/pandas/frame.py

```diff
@@ -4815,6 +4815,13 @@ def to_spark_io(
         index_col: Optional[Union[str, List[str]]] = None,
         **options
     ) -> None:
+        """An alias for :func:`DataFrame.spark.to_spark_io`.
+        See :meth:`pyspark.pandas.spark.accessors.SparkFrameMethods.to_spark_io`.
+
+        .. deprecated:: 3.2.0
+            Use :func:`DataFrame.spark.to_spark_io` instead.
+        """
+        warnings.warn("Deprecated in 3.2, Use spark.to_spark_io instead.", FutureWarning)
```

Comment:
shall we use `DataFrame.spark.to_spark_io`

Reply:
Sounds good! I'll address it, thanks :)